runtime/compiler/il/symbol/J9Symbol.cpp

```diff
@@ -271,6 +271,44 @@ J9::Symbol::getRecognizedField()
       return TR::Symbol::UnknownField;
    }
 
+/**
+ * Return the owning class name of this recognized field.
+ * Return null if this symbol does not have a recognized field.
+ */
+const char *
+J9::Symbol::owningClassNameCharsForRecognizedField(int32_t & length)
+   {
+   length = 0;
```

Comment:
why always clobber length if the real check is supposed to be for a NULL return?

Reply:
Removed this line.